package.json

```diff
@@ -176,7 +176,7 @@
     "postinstall": "node packageScripts/postinstall.mjs",
     "sass:render": "sass-render 'src/**/*.css' 'components/**/*.css' -t ./node_modules/@aurodesignsystem/auro-library/scripts/build/staticStyles-template.js",
     "serve": "web-dev-server --open demo/ --node-resolve --watch",
-    "sweep": "find ./demo ./dist ./src ./components -type f \\( -name \"*.css\" -o -name \"*-css.js\" \\) -delete",
+    "sweep": "rm -rf ./demo/css ./dist | find ./demo ./dist ./src ./-type f \\( -name \"*.css\" -o -name \"*-css.js\" \\) -delete",
```

Comment:
This isn't really a folder is it? `./demo/css`

Reply:
I wasn't sure. It was in the original file, so I restored it:

https://github.com/AlaskaAirlines/auro-input/pull/341/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L179

Should I remove it?

Reply:
Removed, thanks